### ISSUE TITLE
Hammock skips to next morning instead of evening

### ISSUE THREAD

**Issue by FrijDom** [thephoenixlodge/Heavens-of-Sorcery#157]
I've been trying to use the Hammock from Comforts in order to skip through the day in the Misty World (so I can go looking for ores below the mist). However, it seems to be setting the time to the same as beds and other sleeping instruments. I've tested this outside of the pack with just the Comforts mod, and it works as expected. The only possibility I can think of is that one of the other mods that implements a sleeping mechanic is interfering with the Hammock code.

**Comment by ewanm89** [comment]
Yeah, the mod in question, is misty world, and the way that dimension handles sleep.
